opm/input/eclipse/EclipseState/Grid/EclipseGrid.hpp

```diff
@@ -227,6 +228,7 @@ namespace Opm {
         PinchMode m_pinchoutMode;
         PinchMode m_multzMode;
         PinchMode m_pinchGapMode;
+        double    m_pinchMaxEmptyGap = 1e20;
```

Comment:
This is okay I guess, but for these kinds of things we **generally** prefer to retrieve the initial/default value from the [json description](https://github.com/OPM/opm-common/blob/b8ee8d8652a0f76c33e04f8a3b2fcbc3e4d1acec/src/opm/input/eclipse/share/keywords/000_Eclipse100/P/PINCH#L22) instead of hard-coding the value here.  In this case that would require adding something along the lines of
```C++
m_pinchMaxEmptyGap { ParserKeywords::PINCH::MAX_EMPTY_GAP::defaultValue }
```
to every pertinent constructor in `EclipseGrid.cpp`.  If we do go that way, we might also consider refactoring the default settings to the default constructor and have the other constructors delegate to the default one and then just override the values they need.

Reply:
IMHO, I could even have put zero there and it would not matter as if PINCH is specified then ParserKeywords::PINCH::MAX_EMPTY_GAP::defaultValue will be used anyway if nothing is specified there. Hence this is just here for preventing compiler warnings and to not do in every constructor.

Your proposed refactoring would also pull in a/the BIG keyword headers into this header for limited gain. Hence I strongly disagree with that proposal.

Reply:
> IMHO, I could even have put zero there and it would not matter as if PINCH is specified then `ParserKeywords::PINCH::MAX_EMPTY_GAP::defaultValue` will be used anyway if nothing is specified there. 

How so?  There is nothing in the current code (master and this PR) that puts the value anywhere.

Reply:
> [This](https://github.com/OPM/opm-common/blob/ea64c290010ce46cbeb38d8950ecae6ee13cbef8/src/opm/input/eclipse/EclipseState/Grid/EclipseGrid.cpp#L352) should do the trick by getting the default value if PINCH is there but nothing specified, shouldn't it?

Not really, no.  First, that statement applies unit conversion which we arguably should not do for the default value, at least not if we need to identify that the value is defaulted.  More importantly however, that does nothing to handle the case when PINCH **isn't** entered in the deck.  That was the case I wanted to address by having the default constructor assign those default values.